.github/workflows/build-and-test.yml

```
@@ -33,7 +33,7 @@ jobs:
     secrets:
       REMOTE_PROVIDER_TOKEN: ${{ secrets.REMOTE_PROVIDER_TOKEN }}
   publish-allure-report:
-    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}    
+    if: ${{ ((github.event_name == 'push' && github.ref == 'refs/heads/master') || github.event_name == 'pull_request') && secrets.GH_ACCESS_TOKEN != '' }}    
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)